go/erasablekv/erasable_kv_store.go

```diff
@@ -53,18 +54,20 @@ type FileErasableKVStore struct {
 
 var _ ErasableKVStore = (*FileErasableKVStore)(nil)
 
-func NewFileErasableKVStore(g *libkb.GlobalContext) *FileErasableKVStore {
+func NewFileErasableKVStore(g *libkb.GlobalContext, subDir string) *FileErasableKVStore {
 	return &FileErasableKVStore{
 		Contextified: libkb.NewContextified(g),
-		storageDir:   getStorageDir(g),
+		storageDir:   getStorageDir(g, subDir),
 	}
 }
 
 func (s *FileErasableKVStore) filepath(key string) string {
+	key = url.QueryEscape(key)
```

Comment:
Slight preference for creating a new variable here like `escapedKey`, or just inlining the call, rather than reusing the same variable.